### DISCUSSION TITLE
how to store real number with precision size

### DISCUSSION THREAD

**Discussion by PawanGhewande** [simolus3/drift#1596] [Q&A]
I want to store value in db like 1.40 & 1.04, Im currently using RealColumn() so it stores 1.4 and also return the same for both values,

I really don't know how I can solve this issue.

**Comment by frankvollebregt** [comment]
One solution would be to use [type converters](https://drift.simonbinder.eu/docs/advanced-features/type_converters/) to achieve this. You can store a number (under the hood) but convert to and from a given precision value when reading and writing to the database.

**Reply by simolus3** [reply]
I also recommend this approach! Sqlite has not builtin type for fixed-precision numbers, so I think a type converter is the best approach to handle this somewhat transparently.
